internal/component/prometheus/exporter/cadvisor/cadvisor.go

```diff
@@ -37,15 +37,6 @@ type Arguments struct {
 	DisabledMetrics            []string      `alloy:"disabled_metrics,attr,optional"`
 	EnabledMetrics             []string      `alloy:"enabled_metrics,attr,optional"`
 	StorageDuration            time.Duration `alloy:"storage_duration,attr,optional"`
-	ContainerdHost             string        `alloy:"containerd_host,attr,optional"`
```

Comment:
Is there an upstream reason we are removing these settings? Our backwards promise tries to ensure we dont remove settings unless we absolutely have to.

Reply:
I have reverted the removed configuration. Regarding `DisableRootCgroupStats`, I think we should remove it since the code at https://github.com/google/cadvisor/blob/master/container/raw/factory.go#L34 does not expose it for configuration.

Reply:
Hi, could you please help review this PR?

Reply:
It looks like it is used but as a flag which is likely why we expose it as a configuration option. Removing this would still break our promise for backwards compatibility.

Reply:
Should we make this configuration available but mark it as deprecated, so it doesn't take effect? 
Or is there a mistake in how I'm configuring it?